app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -0,0 +1,342 @@
+  ```
+
+5. Start PostgreSQL
+  ```
+  $ sudo systemctl start postgresql
+  ```
+
+**ii. Create a Kong user and database**
+
+The user and database can easily be created via the command line by using PostgreSQL’s command line tool `psql`
+
+1. Change to the `postgres` user
+  ```
+  $ sudo -i -u postgres
+  ```
+
+2. Launch the PostgreSQL command line tool `psql`
+  ```
+  $ psql
+  ```
+
+3. Create the `kong` user and database
+  ```
+  $ psql> CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong';
+  ```
+  > ⚠️Note: Make sure the username and password for the Kong Database are kept safe. We have used a simple example for illustration purposes only.
+
+4. Exit `psql`
+  ```
+  $ psql> \q
+  ```
+
+5. Return to the terminal
+  ```
+  $ exit
+  ```
+
+**iii. Connect Kong to the PostgreSQL database**
+
+Postgres uses `ident` authentication by default, to allow the `kong` user to communicate with the database locally we must change the authentication method to `md5` by modifying the Postgres configuration file. 
+
+
+1. Open the configuration file:
+  ```
+  $ sudo vi /var/lib/pgsql/data/pg_hba.conf
+  ```
+
+2. Change the `IPV4` local connection method to `md5`
+  ```
+    | TYPE | DATABASE | USER | ADDRESS      | METHOD |
+    | # IPv4 local connections:                      |
+    | host | all      | all  | 127.0.0.1/32 |  md5   |
+    | # IPv6 local connections:                      |
+    | host | all      | all  | ::1/128      | ident  |
+  ```
+
+3. Save and exit the file, then restart Postgres
+  ```
+  $ sudo systemctl restart postgresql
+  ```
+
+Now that the database is set up, Kong must be given credentials to access the database. This is done by adding the Postgres user and password information to Kong’s configuration file.
+
+
+1. Make a copy of Kong’s configuration file
+  ```
+  sudo cp /etc/kong/kong.conf.default kong.conf
+  ```
+
+2. Open the new configuration file
+  ```
+  sudo vi /etc/kong/kong.conf
+  ```
+
+3. Search for `pg_password` under `DATASTORE,` uncomment `pg_password` and add the value. 
+  ```
+  #pg_user = kong
+  pg_password = kong
+  #pg_datastore = kong
+  ```
+  > Note: If you used different values for the user and database name, uncomment the pg_user and pg_datastore variables and add the username and datastore values
+
+
+## Step 4. Customize Your Configuration
+Before starting Kong, you can further modify Kong’s configuration file to enable a host of different features:
+
+
+### Enable RBAC
+
+  Kong Enterprise allows applying RBAC to all requests. To enable RBAC, you must set RBAC to `on`, select an authentication plugin for Kong Manager, and configure the [Session plugin](/hub/kong-inc/session/).
+
+  ```
+  enforce_rbac = on
+
+  admin_gui_auth = basic-auth
+
+  admin_gui_session_conf = {"secret":"password", "cookie_secure": false, "cookie_samesite": "off", "cookie_name": "admin_session", "storage": "kong"}
+  ```
+
+  Export the KONG_PASSWORD environment variable:
```

Comment:
Lets definitely keep this here even though we are not going to need it till later. it is painful to have to go back and create a super admin if you dont seed the superadmin correctly. maybe a note to save this password for later when we enable rbac.